db/anilist_users.py

```diff
@@ -0,0 +1,50 @@
+from typing import List, Optional
+
+from sqlalchemy import Integer
+from sqlalchemy.schema import Column
+
+from .base import Base
+
+
+class AnilistUsers(Base):
+    __tablename__ = "anilist_users"
+
+    anilist_id = Column(Integer, default=0)
```

Comment:
`default=None` is better. It is nullable by default

Reply:
Did this, revised other function calls to handle it accordingly.

```py
    def set_anilist_id(self, anilist_id: int):
        if anilist_id == 0:
            self.anilist_id = None
        self.anilist_id = anilist_id
        self._save()
```